bindings/cs/rl.net.native/CMakeLists.txt

```diff
@@ -15,3 +40,10 @@ endif()
 set_target_properties(rl.net.native PROPERTIES POSITION_INDEPENDENT_CODE ON)
 target_include_directories(rl.net.native PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(rl.net.native PUBLIC rlclientlib)
+
+# Enable warnings as errors, crossplatform
+target_compile_options(rl.net.native PRIVATE
+     $<$<OR:$<CXX_COMPILER_ID:Clang>,$<CXX_COMPILER_ID:AppleClang>,$<CXX_COMPILER_ID:GNU>>:
+          -Werror>

```

Comment:
Do we want warnings as errors? While this seems like a good idea it can be a pain

Reply:
Yes. I want to slowly bring this out through the project - after fixing the appropriate warnings on our shipping platforms. Right now it is contained to rl.net.native where I *really* do not want warnings.

Reply:
Funny bit is that I forgot to include warnings-as-errors in Windows. Fixed.